### PR TITLE
UI Integration / ODP get

### PR DESCRIPTION
**Pull request by minotogna**

### CODE REVIEW

File: webapp/pages/OriginalDataPoint/OriginalDataPoint.tsx [hunk 1]
```diff
@@ -25,14 +23,18 @@ const OriginalDataPoint: React.FC = () => {
...
 
   useEffect(() => {
-    dispatch(batchActions([fetch(odpId, countryIso), fetchExtentOfForest()]))
+    dispatch(batchActions([OriginalDataPointActions.fetchODP({ id: odpId }), fetchExtentOfForest()]))
 
     return () => {
-      dispatch(batchActions([CountryActions.fetchCountryStatus(countryIso), clearActive()]))
+      dispatch(
+        // TODO: why OriginalDataPointActions.setODP isn't recognized?
+        // @ts-ignore
```
Comment (minotogna) [inline webapp/pages/OriginalDataPoint/OriginalDataPoint.tsx:35]: @sorja this can be addresses by both of us in the next branches

Comment (minotogna) [reply webapp/pages/OriginalDataPoint/OriginalDataPoint.tsx]: @sorja fixed in next commit.

**Comment by sorja** [comment]
When accessing any ODP I  get error: 
`{"methods": "Other (specify in comments)"} does not have a method named "join"`

File: webapp/store/page/originalDataPoint/slice.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+interface Reducer extends SliceCaseReducers<OriginalDataPointState> {
+  setODP: CaseReducer<OriginalDataPointState, PayloadAction<{ odp: ODP }>>
+}
+
+export const originalDataPointSlice = createSlice<OriginalDataPointState, Reducer>({
+  name: 'originalDataPoint',
+  initialState: {},
+  reducers: {
+    setODP: (state, action) => {
```
[the remark this answers is not in the document]
Comment (minotogna) [reply webapp/store/page/originalDataPoint/slice.ts:17]: it's the same thing

**Comment by minotogna** [comment]
> When accessing any ODP I get error:
> `{"methods": "Other (specify in comments)"} does not have a method named "join"`

because you didn't run the last migration ??

**Pull request merged**